Dockerfile.dev

```diff
@@ -0,0 +1,18 @@
+FROM golang:1.19-alpine
+
+RUN mkdir -p /app
+
+WORKDIR /app/github.com/cryptag/leapchat/miniware
+
+WORKDIR /app/github.com/cryptag/leapchat
```

Comment:
@jimmcgaw I believe we want the 3 lines above into these 2, so that our working directory exists and so we don't have the additional `WORKDIR`:

```dockerfile
RUN mkdir -p /app/github.com/cryptag/leapchat/miniware
WORKDIR /app/github.com/cryptag/leapchat
```

Reply:
Then I think this PR is ready to merge! Is it working well for you locally?

Reply:
Oh yeah there needs to be a bit more done: https://github.com/cryptag/leapchat/milestone/7